mdf.md

```diff
@@ -0,0 +1,82 @@
+#  MVDS Metadata Field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Format](#format)
+    1. [Fields](#fields)
+1. [Usage](#usage)
+    1. [Informational Fields](#informational-fields)
+    2. [Configurational Fields](#configurational-fields)
+
+## Abstract
+
+In this specification, we describe a method to construct both a linear and DAG (Directed Acyclic Graph) based message history that will aid the consistency of [MVDS](./mvds.md). Additionally we explain how data sync can be used for more lightweight messages that do not require full synchronization. This specification extends the [MVDS message](./mvds.md#payloads) to modify the functionality of MVDS.
```

Comment:
What kind of consistency?

Reply:
+1, mapping it to no consistency guarantees; sequential consistency for single writer (linear) and casual consistency (DAG) would be useful. Perhaps as part of its own section below?